api/v2-c/Makefile

```diff
@@ -30,7 +30,7 @@ CFLAGS += -fstack-protector-strong -fPIC
 CFLAGS += -Wall -Werror
 CFLAGS += -D_FORTIFY_SOURCE=2 -O2
 
-SOURCES = $(wildcard */*.c)
+SOURCES = $(wildcard */*.c) $(wildcard workloadapi/security/*.c)
```

Comment:
Does  previous */*.c already includes workloadapi/security/*.c？

Reply:
/.c will only include files in the current directory